test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -463,8 +463,8 @@ public static TheoryData DoesNotRunWhenARequiredAttributeIsMissing_Data
                     {
                         new TagHelperAttributeList
                         {
-                            // This is commented out on purpose: ["asp-href-include"] = "*.css",
-                            ["asp-href-exclude"] = "*.min.css"
+                            // This is commented out on purpose: new TagHelperAttribute("asp-href-include", "*.css"),
```

Comment:
While here (and below), please change these comments to say something like `// Note asp-href-include attribute isn't included`.